app/views/static_pages/home/_how_it_works.html.erb

```diff
@@ -1,34 +1,31 @@
+ <% tiles = [
```

Comment:
@codyloyd Can you move this to [ApplicationHelper](https://github.com/TheOdinProject/theodinproject/blob/redesign/app/helpers/application_helper.rb) and create an  instance variable in the `home` action of `StaticPagesController` to make it available to the views?